src/flatc.cpp

```diff
@@ -124,6 +124,9 @@ static void Error(const std::string &err, bool usage, bool show_exe_name) {
       "  --gen-name-strings Generate type name functions for C++.\n"
       "  --escape-proto-ids Disable appending '_' in namespaces names.\n"
       "  --gen-object-api   Generate an additional object-based API\n"
+      "  --cpp-variant VAR  What C++ variant to generate code for:\n"
+      "                       c++0x - (default) uses push_back and () for constructors\n"
```

Comment:
this help text is a bit specific, since we'll likely extend this functionality well beyond just emplace_back and {}
would be better to say:
c++0x (default): Minimal c++11 functionality at the level of VS2010 / GCC 4.6.2).
c++11: Code for a fully compliant c++11 compiler (VS2015 / GCC 4.?).